packages/shared/src/types/props/epic.ts

```diff
@@ -12,10 +12,13 @@ import {
 import { OphanComponentEvent } from '../ophan';
 import { EpicVariant } from '../abTests';
 
-export interface ArticleCounts {
```

Comment:
This change doesn't affect the API. It just means we can reference the valid article count types elsewhere